src/pint/residuals.py

```diff
@@ -500,6 +500,12 @@ def calc_time_resids(
             )
         return (phase_resids / self.get_PSR_freq(calctype=calctype)).to(u.s)
 
+    def calc_whitened_resids(self):
```

Comment:
Is it better to have a separate method, or `calc_time_resids()` with a flag?

Reply:
I think it should be a separate method since whitened residuals are dimensionless, whereas time resids have the dimentions of time.

Reply:
Then probably add a "See Also" to the other residual methods

Reply:
OK.

Reply:
Updated documentation.